docs/data_schema.md

```diff
@@ -220,41 +217,63 @@ Contains all result data from the static analysis; see description below
 ### `results` object
 
 #### `files`
-List of static analysis results, one per file contained in the analyzed package tarball. Files are enumerated in lexical order. Symlinks or special files such as device files, sockets and pipes are excluded. Each item is corresponds to a SingleResult object in Go; see description below.
+List of static analysis results, one per file contained in the analyzed package tarball. Files are enumerated in lexical order. Symlinks or special files such as device files, sockets and pipes are excluded. Each item is corresponds to a FileResult object in Go; see description below.
 
-### `SingleResult` object
+### `FileResult` object
 
 #### `filename`
 Path of the file, relative to the package archive root
 
-#### `basic` (optional)
-Contains results from the basic static analysis task; this is information that can be obtained without parsing the file. See basic object description below. Field is omitted if the basic analysis task was excluded during the analysis run.
+#### `detected_type`
+Output of `file` command run on the file. Omitted if the `basic` analysis task was not run.
```

Comment:
Might be worth mentioning that it is the file command included by Ubuntu.

Reply:
done